src/main/resources/application.properties

```
@@ -1,7 +1,11 @@
-spring.graphql.graphiql.enabled=true
-spring.graphql.graphiql.path=/graphiql
-spring.datasource.driverClassName=org.postgresql.Driver
-spring.datasource.platform=postgres
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto=none
```

```suggestion
spring.jpa.hibernate.ddl-auto=update
```
se deixar none o servidor nao cria as tabelas das entidades no banco